.github/workflows/dockerhub.yml

```diff
@@ -18,9 +18,14 @@ jobs:
         uses: docker/login-action@v3
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}        
+          
+      - name: Replcae Docker tag
```

Comment:
Looks like Replace has a typo here

Reply:
Good catch! This typo has been corrected!